src/validation/index.ts

```diff
@@ -1,5 +1,19 @@
-import { hasLength, PREFIX_NAME_PATTERN } from 'utils/misc-utils';
-import { PrefixedName_Errors } from './types';
+// Based on pattern taken from
+
+import { PrefixedName_Errors } from 'components/inputs/PrefixedName/types';
+import { hasLength } from 'utils/misc-utils';
+
+//  https://github.com/estuary/animated-carnival/blob/main/supabase/migrations/03_catalog-types.sql
+export const PREFIX_NAME_PATTERN = `[a-zA-Z0-9-_.]+`;
+export const CATALOG_NAME_PATTERN = `^(${PREFIX_NAME_PATTERN}/)+${PREFIX_NAME_PATTERN}$`;
+export const NAME_RE = new RegExp(CATALOG_NAME_PATTERN);
+
+// Based on the patterns connectors use for date time
+// eslint-disable-next-line no-useless-escape
+export const DATE_TIME_PATTERN = `[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z`;
+export const DATE_TIME_RE = new RegExp(`^(${DATE_TIME_PATTERN})$`);
```

Comment:
This was copied from old location. But ended up being invalid because it had an opening `(` but not a closing one. So added the new closing one.